### DISCUSSION TITLE
Erro ao instalar

### DISCUSSION THREAD

**Discussion by saviobraz** [codemagic-ci-cd/codemagic-docs#2679] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Expected file contents.xcworkspacedata in /Users/builder/clone/ios/Runner.xcworkspace, but no such file exists
Mas existe o arquivo Runner.xcworkspace

### Expected results

preciso compilar e publicar na apple store

### Actual results

Expected file contents.xcworkspacedata in /Users/builder/clone/ios/Runner.xcworkspace, but no such file exists
preciso de ajuda, desde já obrigado.

### Build id (optional)

66b9fb36765b49b71bea564e

**Comment by dtrdic** [comment]
Hi @saviobraz, can you please check if the folder structure of your app looks like [this](https://github.com/codemagic-ci-cd/codemagic-sample-projects/tree/main/flutter/flutter-android-and-ios-yaml-demo-project)?
Do you have `xcworkspace` file locally, perhaps you didn't commit it to git? Have you tried to run a build locally?

**Comment by saviobraz** [comment]
Refiz a estrutura com referencia a este link
https://github.com/codemagic-ci-cd/codemagic-sample-projects/tree/main/flutter/flutter-android-and-ios-yaml-demo-project/ios,
mas continua o problema, mas não entendo, pois o arquivo
contents.xcworkspacedata   consta no projeto, sendo há 1 mes atras mais ou
menos eu publiquei com sucesso na apple store, sinceramente não estou
entendendo

Em seg., 12 de ago. de 2024 às 10:18, David Maldonado <
***@***.***> escreveu:

**Comment by saviobraz** [comment]
acho que resolvi, era extensão do arquivo, ocultei estensão .xml, por isso
não reconhecia

Em seg., 12 de ago. de 2024 às 11:32, Sávio de Oliveira Braz <
***@***.***> escreveu:

**Reply by dtrdic** [reply, marked as answer]
Nice to hear that, i'll close this issue then. Have a nice day!

Note: translation

> I think I solved it; it was a file extension issue. I had hidden the .xml extension, which is why it wasn’t recognized.
